runtime/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.3.0
```

Comment:
This should be `2.0.1` once https://github.com/nearprotocol/nearcore/pull/3234 is merged